docs/release-notes/8.0.32-24.md

```diff
@@ -32,15 +32,17 @@ increased visibility.
+    The data and the GTIDs backed up by mysqldump were inconsistent when the options --single-transaction and --set-gtid-purged=ON were both used. It was because in between the transaction started by mysqldump and the fetching of GTID_EXECUTED, GTIDs on the server could have increased already. With this fixed, a FLUSH TABLES WITH READ LOCK is performed before the fetching of GTID_EXECUTED to ensure its value is consistent with the snapshot taken by mysqldump.
 
-The MySQL fix also added a requirement when using [--single-transaction](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_single-transaction) and executing FLUSH TABLES WITH READ LOCK for the [RELOAD](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_reload) privilege. ([MySQL bug #109701](https://bugs.mysql.com/bug.php?id=109701), [MySQL bug #105761](https://bugs.mysql.com/bug.php?id=105761))
+    The MySQL fix also added a requirement when using [--single-transaction](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_single-transaction) and executing FLUSH TABLES WITH READ LOCK for the [RELOAD](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_reload) privilege. ([MySQL bug #109701](https://bugs.mysql.com/bug.php?id=109701), [MySQL bug #105761](https://bugs.mysql.com/bug.php?id=105761))
 
-The Percona Server version of the `mysqldump` utility, in some modes, can be used with MySQL Server. This utility provides a temporary workaround for the "additional RELOAD privilege" limitation introduced by Oracle MySQL Server 8.0.32. 
+    The Percona Server version of the `mysqldump` utility, in some modes, can be used with MySQL Server. This utility provides a temporary workaround for the "additional RELOAD privilege" limitation introduced by Oracle MySQL Server 8.0.32. 
 
-For more information, see the Percona Performance Blog [A Workaround for the "RELOAD/FLUSH_TABLES privilege required" Problem When Using Oracle mysqldump 8.0.32](https://www.percona.com/blog/workaround-for-the-reload-flush_tables-privilege-required-problem-when-using-oracle-mysqldump-8-0-32/).
+    For more information, see the Percona Performance Blog [A Workaround for the "RELOAD/FLUSH_TABLES privilege required" Problem When Using Oracle mysqldump 8.0.32](https://www.percona.com/blog/workaround-for-the-reload-flush_tables-privilege-required-problem-when-using-oracle-mysqldump-8-0-32/).
+
+* Percona Server for MySQL has changed the default for [supported DDL column operations](https://dev.mysql.com/doc/refman/8.0/en/innodb-online-ddl-operations.html) from `ALGORITHM=INPLACE` to `ALGORITHM=INSTANT`. 
```

Comment:
I would put the whole context here...

As all most important bugs related to ALGORITHM=INSTANT were fixed by Oracle in MySQL Server 8.0.32, Percona Server for MySQL 8.0.32 returned original(upstream's) behaviour. For supported DDL column operations the default ALTER TABLE algorithm is now INSTANT again. It was previously deliberately changed  to INPLACE in Percona Server for MySQL 8.0.29 for safety reasons (see this blog post for more details https://www.percona.com/blog/percona-xtrabackup-8-0-29-and-instant-add-drop-columns/).

Reply:
Updated